From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Eric Biggers <ebiggers@google.com>
Date: Tue, 11 Aug 2020 15:44:11 -0700
Subject: ANDROID: soc: qcom: rpmh: fix bad merge resolution

... between commit cd49f10d96fb ("FROMLIST: soc: qcom: rpmh: Allow RPMH
driver to be loaded as a module") and upstream commit 681b09123844
("soc: qcom: rpmh: Allow RPMH on ARM SoC").

Fixes: 075d0ec13d0b ("Merge c0842fbc1b18 ("random32: move the pseudo-random 32-bit definitions to prandom.h") into android-mainline")
Cc: John Stultz <john.stultz@linaro.org>
Change-Id: Ida0fefa19bba5f291218ffa794a095b41599d48e
Signed-off-by: Eric Biggers <ebiggers@google.com>
---
 drivers/soc/qcom/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/qcom/Kconfig b/drivers/soc/qcom/Kconfig
--- a/drivers/soc/qcom/Kconfig
+++ b/drivers/soc/qcom/Kconfig
@@ -93,7 +93,7 @@ config QCOM_RMTFS_MEM
 
 config QCOM_RPMH
 	tristate "Qualcomm RPM-Hardened (RPMH) Communication"
-	depends on ARCH_QCOM && ARM64 || COMPILE_TEST
+	depends on ARCH_QCOM || COMPILE_TEST
 	help
 	  Support for communication with the hardened-RPM blocks in
 	  Qualcomm Technologies Inc (QTI) SoCs. RPMH communication uses an
